areas/aks/src/AzureMcp.Aks/AksSetup.cs

```diff
@@ -20,7 +20,7 @@ public void ConfigureServices(IServiceCollection services)
     public void RegisterCommands(CommandGroup rootGroup, ILoggerFactory loggerFactory)
     {
         // Create AKS command group
-        var aks = new CommandGroup("aks", "Azure Kubernetes Service operations - Commands for managing and listing Azure Kubernetes Service (AKS) clusters.");
+        var aks = new CommandGroup("aks", "Azure Kubernetes Service operations - Commands for managing Azure Kubernetes Service (AKS) cluster resources. Includes operations for listing clusters, retrieving cluster configurations, and managing Kubernetes environments in Azure.");
```

Comment:
@vcolin7 how soon can we test these with the vector similarity score script?

We need to be able to verify these string changes are actually having the positive impact we intend.